addons/web/static/tests/mobile/views/view_dialog/select_create_dialog_tests.js

```diff
@@ -0,0 +1,110 @@
+    const LINKED_SALE_ORDER_LINE = "linked_sale_order_line";
+    const PRODUCT_FIELD_NAME = "product_id";
+
+    hooks.beforeEach(() => {
+        target = getFixture();
+        serverData = {
+            models: {
+                [PRODUCT_PRODUCT]: {
+                    fields: {
+                        id: { type: "integer" },
+                        name: {},
+                    },
+                    records: [
+                        {
+                            id: 111,
+                            name: "product_cable_management_box",
+                        },
+                    ],
+                },
+                [SALE_ORDER_LINE]: {
+                    fields: {
+                        id: { type: "integer" },
+                        [PRODUCT_FIELD_NAME]: {
+                            string: PRODUCT_FIELD_NAME,
+                            type: "many2one",
+                            relation: PRODUCT_PRODUCT,
+                        },
+                        [LINKED_SALE_ORDER_LINE]: {
+                            string: LINKED_SALE_ORDER_LINE,
+                            type: "many2many",
+                            relation: SALE_ORDER_LINE,
+                        },
+                    },
+                },
+            },
+            views: {
+                [`${PRODUCT_PRODUCT},false,kanban`]: `
+                    <kanban><templates><t t-name="kanban-box">
+                        <div class="oe_kanban_global_click">
+                            <field name="id"/>
+                            <field name="name"/>
+                        </div>
+                    </t></templates></kanban>
+                `,
+                [`${SALE_ORDER_LINE},false,kanban`]: `
+                    <kanban><templates><t t-name="kanban-box">
+                        <div class="oe_kanban_global_click">
+                            <field name="id"/>
+                        </div>
+                    </t></templates></kanban>
+                `,
+                [`${PRODUCT_PRODUCT},false,search`]: "<search></search>",
+            },
+        };
+        setupViewRegistries();
+    });
+
+    QUnit.module("SelectCreateDialog - Mobile");
+
+    QUnit.test("SelectCreateDialog: clear selection in mobile", async function (assert) {
+        assert.expect(3);
+
+        await makeView({
+            type: "form",
+            resModel: SALE_ORDER_LINE,
+            serverData,
+            arch: `
+                <form>
+                    <field name="${PRODUCT_FIELD_NAME}"/>
+                    <field name="${LINKED_SALE_ORDER_LINE}" widget="many2many_tags"/>
+                </form>`,
+            async mockRPC(route, args) {
+                if (args.method === "create" && args.model === SALE_ORDER_LINE) {
+                    const selectedId = args.args[0][PRODUCT_FIELD_NAME];
+                    assert.equal(selectedId, false, `there should be no product selected`);
```

Comment:
always use strictEqual (here it doesn't matter, but we don't want people to use it as it sometimes does)